libdevcore/Log.cpp

```diff
@@ -89,6 +89,7 @@ void formatter(boost::log::record_view const& _rec, boost::log::formatting_ostre
         _strm << " " EthNavy << _rec[suffix] << EthReset;
 }
 
+std::atomic<bool> g_vmTraceEnabled{false};
```

Comment:
There is also `std::atomic_flag` which is superior to `std::atomic<bool>` unless you need some integer-like features of the `bool` type.

Reply:
Can you make it `static` for static linkage?

Reply:
`atomic_flag` is lower level and superior in case you implement something low-level like new kind of concurrency primitive.

For program logic `atomic<bool>` is superior because it has more simple interface, including conversion to `bool`.

Reply:
It has internal linkage because it's in the unnamed namespace.